climpred/tests/test_compute_dims.py

```diff
@@ -214,6 +214,61 @@ def test_PM_fails_probabilistic_member_not_in_dim(
     )
 
 
+@pytest.mark.parametrize('dim', [['member'], ['member', 'init']])
+@pytest.mark.parametrize('metric', ['crpss', 'crpss_es'])
+def test_hindcast_crpss(hindcast_recon_1d_ym, metric, dim):
+    he = hindcast_recon_1d_ym.isel(lead=[0, 1])
+    actual = he.verify(
+        dim=dim, metric=metric, alignment='same_verif', comparison='m2o'
+    ).squeeze()
+    print('actual.dims', actual.dims)
+    before_dims = he.get_initialized().dims
+    for d in before_dims:
+        if d not in dim:
+            assert d in actual.dims, print(
+                list(before_dims),
+                '-',
+                dim,
+                '!=',
+                list(actual.dims),
+                'but should be',
+                'check',
+                d,
+            )
```

Comment:
You could make these a little more concise with f-strings. `print(f'{list(before_dims)}-{dim} != {list(actual.dims)} but should be check {d}')`

Reply:
thats from debugging. not a check. but still changed